store/datastore/repos_test.go

```diff
@@ -138,6 +139,52 @@ func TestRepos(t *testing.T) {
 			g.Assert(count).Equal(2)
 		})
 
+		g.It("Should Get a Repo List Paginated", func() {
+			_repos := []*model.RepoLite{
+				{FullName: "bradrydzewski/drone"},
+				{FullName: "drone/drone"},
+			}
+			_repo := []*model.Repo{
```

Comment:
perhaps `repoList` instead of `_repo` since it is an array. When I see `_repo` I think it is a single repo